src/ext/default.c

```diff
@@ -103,6 +103,10 @@ static inline double tfIdfInternal(const ScoringFunctionArgs *ctx, const RSIndex
     return 0;
   }
   uint32_t norm = normMode == NORM_MAXFREQ ? dmd->maxFreq : dmd->len;
+  if (norm == 0) {
```

Comment:
What about BM25? Can some fields also be zero there?

Reply:
Seems OK
https://github.com/RediSearch/RediSearch/blob/2.10/src/ext/default.c#L157